integration_tests/models/sql/test_groupby.sql

```diff
@@ -1,8 +1,23 @@
+with test_data as (
+    
+    select
+        'a' as column_1,
```

Comment:
try using the `safe_cast` macro here to cast this as a string field. That should fix any pg/redshift errors